### ISSUE TITLE
Missing getting started documentation

### ISSUE THREAD

**Issue by DawidStankiewicz** [ufna/VaRest#370]
There is no documentation on provided [wiki link ](https://www.notion.so/VaRest-UE4-Plugin-40b98c54fc184033b60a42e0e4753536) or I can't find it. I see only 3 links not related to getting started. This plugin seems promising, but I don't know how to use it.

Where can I find some use case example?

**Comment by DawidStankiewicz** [comment]
I found this video, it's very useful https://youtu.be/B90jnsEJ6E0

**Comment by Oranjoose** [comment]
Yeah, the old link was https://github.com/ufna/VaRest/wiki/Plugin-Installation . Now it just redirects to the main page of the repository. I should have backed up that page, but instead just saved the link for when I needed it again.

**Comment by ganbaaelmer** [comment]
Yes we need documentation about VaRest plugin. Because i still stuck on Parse Json response. i mean i cannot get variable from whole json response. its tricky as hell. because no documentation about it.
My json is:
(https://cdn.discordapp.com/attachments/932532650588135427/952774272265371668/unknown.png)

**Comment by Maxmystere** [comment]
Doing something like this should get you the result you want
![image](https://user-images.githubusercontent.com/24446198/158135396-4c33dcc4-9436-4b61-a54d-d31a77982f35.png)

**Comment by ganbaaelmer** [comment]
> Doing something like this should get you the result you want ![image](https://user-images.githubusercontent.com/24446198/158135396-4c33dcc4-9436-4b61-a54d-d31a77982f35.png)

you are the hero. thank you!
but i fixid it in my api` item_obj = Item.objects(id=contentiin_oid).first()`    i just need to add `.first()` because its mongodb  :)

**Comment by ValkyrieUK** [comment]
Agree with @DawidStankiewicz the lack of documentation is really painful. Especially as someone who is new to UE development. Would like to see something that would walk through step by step on calling API's and such.
